### DISCUSSION TITLE
Plugin Update notification, add possibility of external email

### DISCUSSION THREAD

**Discussion by simbus82** [joomla/joomla-cms#39855] [2. Cold Features]
Many web agencies and businesses that work with lots of websites have particular emails to manage webisites technical notitifications.

In this plugin is a good practice to let admins add emails not related to admin users.

![image](https://user-images.githubusercontent.com/1375730/218453045-1c7dbdb3-ca96-455b-a0f5-8643ffb01ff2.png)

How about? Shall we make it an issue?

**Comment by brianteeman** [comment]
Don't forget that the email contains a link to perform the update that only a super user can use. 

What I don't understand is why you dont simply create a superuser with this special email address. Surely that is the easiest thing to do?

**Reply by simbus82** [reply]
Right argument, thanks!
The problem is that we try, also to regulate access to the site, to use as few super admins as possible and each user account must really be a person, so as not to have "missing" log problems on who does what.
But could this email be sent without that link, with an "IF", if it is a simple email and not an account?

**Reply by brianteeman** [reply]
as we now have mail templates you can customise it as you wish
